docs/instrumentation-libraries.md

```diff
@@ -0,0 +1,99 @@
+# Support for OpenTelemetry Instrumentation Libraries
+
+OpenTelemetry instrumentation are implemented using a variety of techniques.
+Sometimes the instrumentation is built directly into the library being
+instrumented, sometimes the instrumentation requires the addition of a separate
+instrumentation library, and other times it is a hybrid of those approaches. This
+project aims to make using these instrumentation approaches easier and more
+automatic for our users. This requires us to both vet and dynamically enable a
+subset of the instrumentation that is available to the OpenTelemetry community,
+while still providing some flexibility to allow users to manually load and enable
+instrumentation that we do not support out of the box.
+
+## Libraries and our instrumentation support plans
+
+The libraries listed here contain our current thoughts about which libraries should
+include out of the box support, and which libraries we decided to not support. If
+you have a library that you would like to consider adding to this list, please
+submit an issue to request its inclusion.
+
+### Application Frameworks
+
+1. ASP.NET
+1. AspNetCore
+1. Windows Workflow Foundation
+1. OWIN
+1. Azure Service Fabric
+
+### Databases
+
+#### General database frameworks
+
+1. ADO.NET
+1. EntityFramework
+1. Dapper
```

Comment:
Consider adding links, I assume Dapper here refers to https://github.com/DapperLib/Dapper?

Reply:
Yes, and that's a good idea.